src/Vehicle.cpp

```diff
@@ -66,6 +66,32 @@ void Vehicle::draw(sf::RenderTarget& target, const sf::RenderStates& /*states*/)
     rectangle.setRotation(getRotation());
     rectangle.setFillColor(sf::Color::Yellow);
     target.draw(rectangle);
+
+    auto fl_wheel = sf::RectangleShape(rectangle.getSize() / 4.f);
+    fl_wheel.setOrigin(fl_wheel.getSize() / 2.f);
+    fl_wheel.setPosition(getPosition() + sf::Vector2f(rectangle.getSize().x / 2.f, -rectangle.getSize().y / 2.f));
```

Comment:
Maybe it's worth creating a local variable for position & half size just so these lines are less long.

Reply:
Good point. I removed some redundant code. I could simplify it further but don't want to over-engineer anything right now.